### PR TITLE
Rcg/feature branch example

### PR DESCRIPTION
**Pull request by RachaelGomez**

### CODE REVIEW

**Comment by cescarez** [comment]
:closed_umbrella::closed_umbrella::closed_umbrella::closed_umbrella::closed_umbrella:
:cookie::cookie::cookie::cookie::cookie::cookie::cookie::cookie::cookie::cookie:

File: README.md [hunk 1]
```diff
@@ -4,7 +4,18 @@ Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor i
...
+A full commitment's what I'm thinking of
+You wouldn't get this from any other guy
+I just wanna tell you how I'm feeling
+Gotta make you understand
+Never gonna give you up
+Never gonna let you down
+Never gonna run around and desert you
+Never gonna make you cry
+Never gonna say goodbye
+Never gonna tell a lie and hurt you
```
Comment (marks214) [inline README.md:18]: Love this song

Comment (cescarez) [reply README.md:18]: lol ship it

**Comment by marks214** [comment]
Awesome job!!! :)

File: rachaels-secrets.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+hey. you shouldn't be looking here! They're my secrets!!!
```
Comment (cescarez) [inline rachaels-secrets.txt:1]: :neckbeard:

**Review by marks214 — APPROVED** [review]
✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅

**Pull request merged**